components/common/ConnectWallet/WalletMenuItem.vue

```diff
@@ -105,25 +133,26 @@ const onClickWallet = (wallet: BaseDotsamaWallet): void => {
 const setWallet = async (wallet: BaseDotsamaWallet) => {
   emit('setWallet', wallet)
   // web3 wallet connect logic here & show accountSelect, async or not?
-  await wallet.enable()
+  isAuth.value = true
 
+  await wallet.enable()
   // init account
-  wallet
-    .getAccounts()
-    .then((data) => {
-      walletAccounts.value = data ? data.map(formatAccount) : []
-      localStorage.setItem('wallet', wallet.extensionName)
-    })
-    .catch((e) => {
-      $consola.error('init account error', e)
-    })
+  try {
+    const data = await wallet.getAccounts()
+    walletAccounts.value = data ? data.map(formatAccount) : []
+    localStorage.setItem('wallet', wallet.extensionName)
```

Comment:
`useLocalStorage` may help here

Reply:
yep, but is it a better idea that this pr do not change the previous usage here?
i can raise another pr to resolve this if it's needed

Reply:
Aight